### PR TITLE
Package json lint test

### PR DESCRIPTION
**Pull request by saket2403**
### Summary
Add package json lint test for all packages

Closes #

### Deployment Link

https://terra-framework-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -85,6 +89,8 @@
     "@babel/preset-react": "^7.0.0",
     "@cerner/browserslist-config-terra": "^3.1.0",
     "@cerner/eslint-config-terra": "^5.0.0",
+    "@cerner/package-json-lint": "^1.0.0",
+    "@cerner/package-json-lint-config-terra": "^1.0.0",
     "@cerner/jest-config-terra": "^1.0.0",
```
Comment (benbcai) [inline package.json:94]:
```suggestion
    "@cerner/jest-config-terra": "^1.0.0",
    "@cerner/package-json-lint": "^1.0.0",
    "@cerner/package-json-lint-config-terra": "^1.0.0",
```

File: packages/terra-abstract-modal/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Added package json lint test.
+
 * Changed
   * Updated WDIO screenshots due to webpack-config-terra sass upgrade.
```
Comment (benbcai) [inline packages/terra-abstract-modal/CHANGELOG.md:9]:
```suggestion
* Changed
  * Updated WDIO screenshots due to webpack-config-terra sass upgrade.
  * Added package json lint test.  
```

File: packages/terra-aggregator/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
```
Comment (benbcai) [inline packages/terra-aggregator/CHANGELOG.md:5]: Duplicate Changed section. Same for the other changelogs.

File: packages/terra-application-links/CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,7 @@
 * Changed
   * Updated WDIO screenshots due to webpack-config-terra sass upgrade.
   * Updated WDIO snapshots due to terra-functional-testing v2 upgrade.
+  * Added package json lint test.
```
Comment (benbcai) [inline packages/terra-application-links/CHANGELOG.md:8]: This should be in the `Added` section in all the changelogs.

File: package.json [hunk 5]
```diff
@@ -29,6 +29,9 @@
     "rules": {
       "react/forbid-prop-types": "off"
     }
+  },
+    "package-json-lint": {
+    "extends": "@cerner/package-json-lint-config-terra"
   },
```
Comment (benbcai) [inline package.json:35]:
```suggestion
  "package-json-lint": {
    "extends": "@cerner/package-json-lint-config-terra"
  },
```

**Review by benbcai — APPROVED** [review]

**Comment by benbcai** [comment]
@saket2403 Can this be merged or is it blocked?

**Comment by saket2403** [comment]
> @saket2403 Can this be merged or is it blocked?

This PR is ready to be merged. @benbcai

**Pull request merged**